manager/manager.go

```diff
@@ -783,6 +790,35 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// Enables overwriting an existing containerData/Handler object for a given containerName.
+// can't use createContainer as it just returns if a given containerName has a handler already
+// Ex: rkt handler will want to take priority over the raw handler, but the raw handler might be created first
+
+// Only allow raw handler to be overriden
+func (m *manager) overrideContainer(containerName string, watchSource watcher.ContainerWatchSource) error {
```

Comment:
One thing I was pondering was adding flags to the event struct, and one would be overridable.  i.e. instead of having manager have to know which watcher sources can override others.

i.e. we don't want the raw handler overwriting itself (i.e. via housekeeping this will happen a lot).